v3/handwired/dactyl_lightcycle/dactyl_lightcycle.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "Dactyl Lightcycle",
+  "vendorId": "0x444D",
```

Comment:
in QMK source this is 0xFEED.

Reply:
[qmk/qmk_firmware#21462](https://github.com/qmk/qmk_firmware/pull/21462) corrects this

Reply:
^ will wait on this before merging 👍